build.sbt

```diff
@@ -36,6 +36,8 @@ lazy val core: Project = (project in file("clients/core"))
       "com.softwaremill.sttp.tapir" %% "tapir-netty-server" % tapirVersion,
       "com.softwaremill.sttp.tapir" %% "tapir-prometheus-metrics" % tapirVersion,
       "com.softwaremill.sttp.tapir" %% "tapir-json-circe" % tapirVersion,
+      "org.typelevel" %% "cats-effect" % "3.5-6581dc4",
```

Comment:
latest stable is 3.4.4, let's use that :)

Reply:
done